ckanext/recombinant/templates/recombinant/snippets/data_dictionary.html

```diff
@@ -44,24 +45,43 @@ <h3>{{ _("Data Dictionary") }}</h3>
         {% call dictionary_field(_('Obligation')) %}
           {{ _('Primary key') if field.datastore_id in chromo.datastore_primary_key
           else _('Required') if field.excel_required
-          else '' }}
+          else field.get(h.lang()) if field.get('obligation') is mapping else _(field.get('obligation', '')) }}
```

Comment:
```suggestion
          else h.recombinant_language_text(field.get('obligation', '')) }}
```

Reply:
I think it was just not working for some reason in the macro before. But this works fine now